src/screens/LoginPage/LoginPage.tsx

```
@@ -303,6 +302,8 @@ const loginPage = (): JSX.Element => {
         setItem('IsLoggedIn', 'TRUE');
         setItem('name', user.name);
         setItem('email', user.emailAddress);
+        setItem('role', user.role)
```

ESLint

[error] Insert `;`

(prettier/prettier)